internal/devicescale/impl_desktop_x.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !android && !ios && !js && !(darwin || windows)
```

Comment:
Isn't this

```
!android && !darwin && !js && !windows
```

`!dawin` already indicates `!ios`

Reply:
Hmm, the current `!(darwin || windows)` is less readable to me... Let's keep this simple as much as possible. As there are other files for mobiles, I'm fine with my suggestion.